Automattic/wp-calypso
# Social Previews | Update Google Search preview

Related to 1203820933396971-as-1204281809108929/f

## Proposed Changes

* Rename `SearchPreview` to `GoogleSearchPreview`
* Convert it to TypeScript
* Update `GoogleSearchPreview` component to reflect the latest design of Google searches
* Create `LinkedInPreview` component in `social-previews` package
* Update the Google preview props used in "Website Meta" preview at `https://wordpress.com/marketing/traffic/:site`


### Testing

- Make sure you have a self-hosted site connected to Jetpack, with a subscription that includes Social.
- Spin up Calypso, by running this branch locally or using a live link below.
- Go to `/marketing/traffic/:site`
- Scroll down to "Website Meta" section
- Click on "Show Previews" to open the previews modal
- Confirm that the Google Search preview looks as expected.
- Also confirm that the Jetpack PR works as expected - https://github.com/Automattic/jetpack/pull/30352

| BEFORE | AFTER |
|--------|--------|
| <img width="942" alt="Screenshot 2023-04-28 at 2 45 06 PM" src="https://user-images.githubusercontent.com/18226415/235112755-06fe0ef2-bfd0-4c3f-abd9-8f8a35d39923.png"> | <img width="945" alt="Screenshot 2023-04-28 at 2 45 48 PM" src="https://user-images.githubusercontent.com/18226415/235112843-c9c62591-6c7f-4639-a590-0e6e14afde81.png"> | 


## Pre-merge Checklist





---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204281809108929
